services/wfm/src/main/java/org/openkilda/wfm/topology/network/NetworkTopologyConfig.java

```diff
@@ -102,6 +102,9 @@ default String getKafkaSwitchManagerRequestTopic() {
     @Default("2")
     int getCountSynchronizationAttempts();
 
+    @Key("port.antiflap.stats.dumping.interval.seconds")
+    int getPortAntiFlapStatsDumpingInterval();
```

Comment:
we can add `@Default("60")` just in case

Reply:
Added